packages/api/src/main.ts

```diff
@@ -5,8 +5,16 @@ import { config } from './config';
 
 async function bootstrap() {
     const app = await NestFactory.create(AppModule);
+
+    const origin = config.cors.origin
+        ? config.cors.origin.map((str) => {
+              const strEscaped = str.replace(/\./g, '\\.');
```

Comment:
Why not `encodeURIComponent()`?
I think its more general

Reply:
`encodeURIComponent()` don't escape `.`, but it's required for RegExp
<img width="324" alt="image" src="https://github.com/a-la-fois/a-la-fois/assets/24812993/07ea6453-1aa0-4740-990a-ff7e3309362f">